frontend/src/scenes/insights/FunnelInsight.tsx

```diff
@@ -26,7 +26,7 @@ export function FunnelInsight(): JSX.Element {
                 onSaveCohort={() => {
                     setCohortModalVisible(true)
                 }}
-                showModalActions={filters.aggregation_group_type_index != undefined}
+                showModalActions={filters.aggregation_group_type_index == undefined}
```

Comment:
what's going on here? O.o - did we intend to show actions earlier for groups? or bug?

Reply:
This was a bug but it wasn't doign any damage because the check inside the component didn't allow anything funnel insight related to show modal actions anyway